### DISCUSSION TITLE
Change of "WG_HOST=🚨YOUR_SERVER_IP"

### DISCUSSION THREAD

**Discussion by Toebus** [wg-easy/wg-easy#559] [General]
Noob here.
First of all it works like a charm :)
 
But lets say my public IP changes how do i enter the config files and edit that? (if i even can)
I have tried: docker run <Contrainer ID>  -e WG_HOST="10.2.X.X" 
Sorry if it's to much of a beginner question :)

**Comment by Pleonazm** [comment]
Use DNS name instead of IP.
If this is the same private subnet, you may even use local mDNS resolution.
You must have Avahi (Linux) or Bonjour (Apple/Windows) installed.
Then use name_of_host.local as your WG_HOST.
Or simply recreate the container, all files are on your disk already in a volume.
